### PR TITLE
Add a description of how data is processed to the privacy policy

### PR DESCRIPTION
**Pull request by rougetimelord**
Responding to #321. 
Maybe they'll be happy if we say that we don't send anything over the network?? Idk. I think the next step would be to add a consent popup just for firefox lol...

### CODE REVIEW

**Comment by cooljeanius** [comment]
@kheina *just* changed one of these lines in #322...

**Comment by rougetimelord** [comment]
> @kheina _just_ changed one of these lines in #322...

I saw, I think it's a good idea to split the how data is processed off into it's own section separate from the purposes just so it's clearer

**Comment by cooljeanius** [comment]
> > @kheina _just_ changed one of these lines in #322...
> 
> I saw, I think it's a good idea to split the how data is processed off into its own section separate from the purposes just so it's clearer

I mean, I guess, but I still want to hear @kheina's opinion first before I approve...

File: privacy.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ This privacy policy ("policy") will help you understand how Blue Blocker collect
 
 This policy may change from time to time and we will do our best to promptly update you on changes. If you want to make sure that you are up to date with the latest changes, we advise you to visit the version of this policy hosted on [GitHub](https://github.com/kheina-com/Blue-Blocker/blob/main/privacy.md).
 
+This version of the Privacy Policy was last updated on 9/6/2024.
```
Comment (cooljeanius) [inline privacy.md:7]: Do we want to clarify that this is an American-style date? i.e., September 6th, rather than June 9th...

Comment (rougetimelord) [reply privacy.md]: Yeah probably lol

**Review by cooljeanius — APPROVED** [review]

**Pull request merged**